lang/en.js

```diff
@@ -32,6 +32,9 @@ export default {
 	"components.form-element.input.url.typeMismatch": "URL is not valid",
 	"components.form-element.valueMissing": "{label} is required.",
 	"components.form-error-summary.errorSummary": "{count, plural, one {There was {count} error found in the information you submitted} other {There were {count} errors found in the information you submitted}}",
+	"components.input-color.backgroundColor": "Background Color",
+	"components.input-color.foregroundColor": "Foreground Color",
```

Comment:
This uncovered the fact that core doesn't support the `en-CA` or `en-GB` language packs. Will deal with that separately.